core/engine_test.go

```diff
@@ -914,54 +914,77 @@ func TestEmittedMetricsWhenScalingDown(t *testing.T) {
 	assert.InDelta(t, 3.35, durationSum/(1000*durationCount), 0.25)
 }
 
-func TestMinIterationDuration(t *testing.T) {
+func TestMetricsEmission(t *testing.T) {
 	t.Parallel()
```

Comment:
Given the failures in appveyor, can you try to remove `t.Parallel` here and below in the sub test cases ? Maybe we an put an 
```golang
if !isWindows{
    t.Parallel()
}
```
later

Reply:
Yeah, it passed on b6650a55, but that might've been a fluke, not sure...

I'm not a fan of conditionally disabling tests or test features like this. Do we know the root cause of this (AppVeyor, Windows, Go, or something in k6 or the test code itself)? Should we look into it?

Reply:
I reran it once and it worked again .. reruning it again ;)
I also dislike it's why we want to use github actions/azure in order to ... somewhat remove the possibility that appveyor is just giving us overloaded machines all the time. 
Given that it's flaky I would argue that while we possibly can make it less flaky (by increasing the time.* variables) it works all the time in CircleCI so ...
Also we are not disabling it, we are just not running it in parallel with a lot of other tests. 

To be honest we do have this problem that especially in some packages we have 100+ tests all of which we intend on running in parallel which is probably not great from performance perspective. I would need to look into whether golang just start running all the 100+ tests or does it start just GOMAXPROCS count ? Because if it's the first one ... if we have 100+ tests that will be very problematic .. especially with time sensitive tests

Reply:
it [looks like it is up](https://golang.org/src/testing/testing.go?s=27209:27231#L999) to GOMAXPROCS ... which is pretty good in our case, so I am going with appveyor is giving us overloaded machines ... :(

Reply:
Thanks for looking into it @MStoykov. Indeed, the switch to GH/Azure would standardize things and remove the uncertainty. I'm fine with this workaround for now.